Cargo.toml

```diff
@@ -5,7 +5,7 @@ debug = 0
 
 [workspace]
 default-members = ["git-branchless"]
-members = ["git-branchless", "git-branchless-lib"]
+members = ["git-branchless", "git-branchless-lib", "git-record"]
```

Comment:
I find it easiest if I put all crates in a `crates/` and for this I do `"crates/*"`

Reply:
I'll keep it in mind; for now, I don't want to increase the directory nesting level too much if I can avoid it.